app/helpers/common/priceFormatter.js

```diff
@@ -1,16 +1,15 @@
-export const formatPrice = (priceValue) => {
-  const formattedPrice = priceValue.toLocaleString(undefined, {
-    minimumFractionDigits: 3, maximumFractionDigits: 3,
-  });
+export const formatNumber = ({
+  value, useGrouping = true, minimumFractionDigits = 0, maximumFractionDigits = 4,
+}) => (
+  parseFloat(value)
+    .toLocaleString(
+      undefined, { minimumFractionDigits, maximumFractionDigits, useGrouping },
+    )
+);
 
-  const truncatedTo2dp = formattedPrice.substring(0, formattedPrice.indexOf('.') + 3);
-
-  return truncatedTo2dp;
-};
-
-export const formatDecimal = (priceValue) => {
-  if ((priceValue || {}).toString().includes('.') && priceValue.toString().split('.')[1].length < 3) {
-    return parseFloat(priceValue).toFixed(2);
+export const formatPrice = (value, maximumFractionDigits = 4, minimumFractionDigits) => {
```

Comment:
I ordered the params to minimize code on use assuming that we are more likely to want to display to 2dp than display hole numbers for a price.

I will change to pass an object.

Reply:
The reason for `minimumFractionDigits ` not being a default parameter is that a price which is desired to be a whole number but is not one, it will be set to `2` in the function body. It is a bit confusing to read.